Tests/MapboxMapsTests/Location/Puck/Mocks/MockPuck.swift

```diff
@@ -21,4 +21,15 @@ final class MockPuck: Puck {
             setPuckBearingSourceStub.call(with: newValue)
         }
     }
+
+    let setPuckBearingEnabledStub = Stub<Bool, Void>()
+    var puckBearingEnabled: Bool {
+        get {
+            fatalError("unimplemented")
+        }
+        set {
+            setPuckBearingEnabledStub.call(with: newValue)
+        }
+    }
```

Comment:
We have a convenience you can use for this:

```
@Stubbed var puckBearingEnabled: Bool = true
```

Then, to access the stub for the setter, you write `puck.$puckBearingEnabled.setStub`

Reply:
Feel free to refactor the other properties in this mock class as well if you have time.

Reply:
This is implemented using Swift property wrappers: https://docs.swift.org/swift-book/LanguageGuide/Properties.html#ID617